sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -218,9 +218,7 @@ private[hive] class SparkExecuteStatementOperation(
                   execute()
                 }
               } catch {
-                case e: HiveSQLException =>
-                  setOperationException(e)
-                  log.error("Error running hive query: ", e)
```

Comment:
for all throwables, they are all logged here https://github.com/apache/spark/pull/29189/files#diff-72dcd8f81a51c8a815159fdf0332acdcL312  once and rethrown again wrapped as `HiveSQLException` which then captured and logged here again

Reply:
Ah, I see. Nice catch.